tests/conftest.py

```diff
@@ -0,0 +1,9 @@
+import pytest
+from src.config import load_config
+from src.log_utils.load_eval_logs import get_log_paths, load_evaluation_logs
+
+
+@pytest.fixture
```

Comment:
Otherwise the fixture is recreated for all tests.

```suggestion
@pytest.fixture(scope="session")
```